app/src/main/res/values/strings.xml

```diff
@@ -26,15 +26,15 @@
     <string name="settings__theme">Theme</string>
     <string name="settings__category_phone">Phone</string>
     <string name="settings__contact">Contact</string>
-    <string name="settings__at_least_one_contact_requried">Your phone needs at least one contact enabled.</string>
+    <string name="settings__at_least_one_contact_requried">At least one contact is needed.</string>
 
-    <string name="edit_contact__unable_to_open_camera_app">Unable to open camera app</string>
-    <string name="edit_contact__confirm_delete_title">Confirm delete</string>
-    <string name="edit_contact__confirm_delete_message">Are you sure you want to remove this contact? Their image and all recordings will be lost.</string>
-    <string name="edit_contact__confirm_delete_recording_message">Are you sure you want to remove this recording?</string>
+    <string name="edit_contact__unable_to_open_camera_app">Could not open camera</string>
+    <string name="edit_contact__confirm_delete_title">Confirm deletion</string>
+    <string name="edit_contact__confirm_delete_message">Remove this contact including their image and recordings?</string>
+    <string name="edit_contact__confirm_delete_recording_message">Remove this recording?</string>
     <string name="edit_contact__name">Name</string>
-    <string name="edit_contact__suggestions">Tips: Centre your face in the photo. Record sounds that mimic how you respond to your baby in normal speech.</string>
+    <string name="edit_contact__suggestions">Tips: Centre your face in the photo. Record sounds like normally responding to your baby.</string>
 
-    <string name="contact_list__description">Add relatives and friends then disable the default contacts for a more personal phone experience. Tap your custom personalised to edit. Defaults cannot be edited, just disabled.</string>
```

Comment:
"Tap your custom personalised to edit" means "the contact"?

Reply:
Whoops, that is just not a complete sentence at all :fearful: